test/test_gromov.py

```diff
@@ -37,9 +38,10 @@ def test_gromov(nx):
     C2b = nx.from_numpy(C2)
     pb = nx.from_numpy(p)
     qb = nx.from_numpy(q)
+    G0b = nx.from_numpy(G0)
 
-    G = ot.gromov.gromov_wasserstein(C1, C2, p, q, 'square_loss', verbose=True)
-    Gb = nx.to_numpy(ot.gromov.gromov_wasserstein(C1b, C2b, pb, qb, 'square_loss', verbose=True))
+    G = ot.gromov.gromov_wasserstein(C1, C2, p, q, 'square_loss', G0=G0, verbose=True)

```

Comment:
do not put G0 in all the tests we want to check that the function works also when not set